source/playbooks/overview.rst

```diff
@@ -18,26 +18,52 @@ Overview
+
+Each playbook represents a recurring outcome or specific goal that your teams collaborate on to achieve, such as service outage recovery or customer onboarding. 
+
+Playbooks are made up of:
+
+- **Checklists**: The list of tasks to be completed for the run. `Checklists <https://docs.mattermost.com/playbooks/customize-a-playbook.html#make-checklists>`_ can be edited ad-hoc during a run.
+- **Templates**: `Templates <https://docs.mattermost.com/playbooks/overview.html#templates>`_ for frequently-used actions such as updates and reminders. You can create your own templates or use default ones.
+- **Actions**: `Automation options <https://docs.mattermost.com/playbooks/customize-a-playbook.html#actions>`_ for inviting members, creating webhooks, editing welcome messages, and more.
+- **Permissions**: Manage :doc:`permissions </playbooks/playbook-permissions>` at a channel and a playbook level.
+
+Teams run a playbook every time they want to orchestrate people, tools, and data to achieve that outcome as quickly as possible while providing visibility to stakeholders. 
+
+For participants, it prescribes processes such as task checklists, status updates, and retrospective reports. For integrated tools, it configures the triggers to perform automated actions. And for stakeholders, it provides a single pane of glass for visibility into each run as well as aggregate insights over time.
 
 Playbooks also allow teams to incorporate learnings from the retrospective to tweak and improve the playbook with every iteration.
 
-Every reported incident or new checklist results in a newly-created, public or private incident channel exposing a custom user interface for collaboration, resolution, and feedback.
+What’s a playbook?
+^^^^^^^^^^^^^^^^^^
+
+A playbook is a checklist of the tasks that make up your processes. These tasks can range from at-mentioning a team member, to using a slash command to start a Zoom call. Tasks can be assigned to users, they can be given due dates.
+
+There are other parts of a playbook, such as automation settings, and metrics. But the very first thing you’ll want to set up is a checklist.
+
+Each time you use the process you’ve documented, such as onboarding a new customer, the playbook is used to start a run - a discrete single use of the process - and that run is captured in a dedicated channel. 
+
+Setting up a playbook includes configuring how the playbook manages the creation of its channel as well as how stakeholders are notified.
+To open a playbook and view its statistics, select the playbook name. To begin a run using a specific playbook, select **Run** beside that playbook’s name.
+
+Templates
+~~~~~~~~~
+
+Creating a playbook from scratch can be daunting, even if you have the process mapped out. One way to get started quickly is to use one of the pre-configured templates available. These templates are populated with content and settings to provide guidance and are customizable.
```

Comment:
nit: `Creating a playbook from scratch can be daunting, even if you have the process mapped out.` sounds like a poor review of our own product :D

It might be true, but can we consider rephrasing this? Also, the templates don't really help if you have the process mapped out: they are more for inspiration in deciding to start modelling processes you might already have. It's pretty unlikely the templates we ship by default will be useful out of the box.

Reply:
Really good point. 

How about:

Creating your playbook from scratch doesn't have to be daunting. If you already have a workflow in mind, the best way to get started is to open the playbook editor and create a checklist based on your workflow. You can test your process as you go and refine it as necessary. 

If you don't know where to start, use one of our templates to help guide you.